test/test_util.py

```diff
@@ -115,7 +115,7 @@ def popen_fail(*args, **kwargs):
         with self.assertRaises(subprocess.CalledProcessError) as exc_context:
             util.command_output([b"taga", b"\xc3\xa9"])
         self.assertEqual(exc_context.exception.returncode, 1)
-        self.assertEqual(exc_context.exception.cmd, b"taga \xc3\xa9")
+        self.assertEqual(exc_context.exception.cmd, 'taga \xc3\xa9')
```

Comment:
It depends on what our goal is for paths. If we plan on moving to pathlib (or otherwise surrogateescaped strings), then I think we should continue treating them as strings here as well. 

if we do that, then we can simplify all sorts of code while keeping our complexity localized.